cdc/sink/sink.go

```diff
@@ -39,6 +39,8 @@ type Sink interface {
 	// FIXME: some sink implementation, they should be.
```

Comment:
```suggestion
	// FIXME: some sink implementation is not thread-safe, but they should be.
```

Please correct similar comments in this file, thanks!